Expecto/Expecto.fs

```diff
@@ -17,10 +17,30 @@ with
     { sourcePath = ""
       lineNumber = 0 }
 
+type FsCheckConfig =
+  {
+    maxTest: int
+    startSize: int
+    endSize: int
+    replay: (int*int) option
+    arbitrary: Type list
+  }
+  static member Default =
```

Comment:
Can we name this semantically after what it intends to be the default, rather than stating that *something* is the default? What if we change the defaults? Then it would be nice to just add another property that has a slightly different configuration and makes that clear from its name. That way, a consumer won't be unexpectedly broken in future versions if we *do* change these defaults.